docs/sources/setup/install/helm/install-monolithic/_index.md

```diff
@@ -81,9 +81,6 @@ If you set the `singleBinary.replicas` value to 2 or more, this chart configures
             schema: v13
         storage:
           bucketNames:
```

Comment:
You should also remove line 83 so we don't have two `bucketNames` keys

Reply:
Ah, good catch, thanks!